src/Tribe/Admin/Move_Ticket_Types.php

```diff
@@ -25,7 +25,7 @@ public function move_tickets_dialog_vars( array $vars ) {
 		}
 
 		return array_merge( $vars, array(
-			'title'    => __( 'Move Ticket Types', 'event-tickets' ),
+			'title'    => esc_html( sprintf( __( 'Move %s Types', 'event-tickets' ), tribe_get_ticket_label_singular( 'move_ticket_type_title' ) ) ),
```

Comment:
Can you verify escaping isn't already done in usage of the dialog vars?

Reply:
It does not for `title`: https://github.com/moderntribe/event-tickets/blob/feature/130897-filterable-ticket-labels/src/Tribe/Admin/Move_Tickets.php#L88 passes to https://developer.wordpress.org/reference/functions/iframe_header/ which does not escape.

However, `title` does get escaped in our own template: https://github.com/moderntribe/event-tickets/blob/feature/130897-filterable-ticket-labels/src/admin-views/move-tickets.php#L12

So I will leave as-is unless you say otherwise.